cli/import-jdl.js

```diff
@@ -268,6 +268,12 @@ class JDLProcessor {
     getConfig() {
         if (jhiCore.FileUtils.doesFileExist('.yo-rc.json')) {
             logger.info('Found .yo-rc.json on path. This is an existing app');
+            if (!Object.prototype.hasOwnProperty.call(this.options, 'interactive')) {
```

Comment:
why not just have 
```javascript
this.options.interactive = true;
this.options['skip-git'] = true;
```

Reply:
I thought that we want it as the default behavior but if the flags are already defined, that means the user want to behave to a certain way.

Reply:
Well, as there are only two possible values and we always set them to true...

Reply:
I am not sure to understand

Reply:
Ah!I think I've misunderstood. Your code set the values to `true` if the values aren't set.
I didn't think of that.

However, why not did you write `Object.prototype.hasOwnProperty.call` instead of `MY_OBJECT.hasOwnProperty`?

Reply:
Yes ! But I have discussed with @PierreBesson about whether we should enforce these flags and it seems that we maybe should enforce the flags since they were meant for the jhipster command and not import-jdl ?

> However, why did you write Object.prototype.hasOwnProperty.call instead of MY_OBJECT.hasOwnProperty?

Just to be sure to use the right function, since an object can define a field hasOwnProperty.
(and eslint does not like [it](https://eslint.org/docs/rules/no-prototype-builtins))

Reply:
I'm not okay with this, the default is non-interactive mode and if you want to run in interactive mode you specify the flag